pepper-apis/src/test/java/org/broadinstitute/ddp/util/TestingUserUtil.java

```diff
@@ -58,18 +58,14 @@
                                                                   String studyGuid) throws Auth0Exception {
 
         var mgmtClient = Auth0ManagementClient.forStudy(handle, studyGuid);
-        Auth0Util auth0Util = new Auth0Util(mgmtClient.getDomain());
-        String mgmtToken = mgmtClient.getToken();
-        final Auth0Util.TestingUser testUser = auth0Util.createTestingUser(mgmtToken);
+        Auth0Util.TestingUser testUser = Auth0Util.createTestingUser(mgmtClient);
```

Comment:
I did not add retries to every single Auth0 call, but only the ones that frequently fail in CircleCI, such as this create user call.